#!/bin/sh /usr/share/dpatch/dpatch-run

## tvonscreen-1.3.31 patch by Thomas Günther <tom@toms-cafe.de>
## http://toms-cafe.de/vdr/download/tvonscreen-1.3.31.diff
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Changes for VDR >= 1.3.31. This patch is only necessary if enAIO-2.6
## DP: or LifeBuffer patch is used.

@DPATCH@
--- tvonscreen-0.7.0/timer.h
+++ tvonscreen-0.7.0/timer.h
@@ -11,10 +11,12 @@
 #define TIMER_H
 
 #include <vdr/plugin.h>
+#include <vdr/menu.h>
 #include <time.h>
 #include "gfxtools.h"
 #include "config.h"
 
+#if VDRVERSNUM < 10331
 #if VDRVERSNUM >= 10300
 class cMenuEvent : public cOsdMenu {
 private:
@@ -48,6 +50,7 @@
   virtual ~cMenuEditTimer();
   virtual eOSState ProcessKey(eKeys Key);
   };
+#endif
 
 class tvOcMenuEvent : public cMenuEvent {
 public:
